integration-tests/tx-rollup-l2-address.spec.ts

```diff
@@ -57,20 +59,84 @@ CONFIGS().forEach(({lib, setup, protocol, txRollupAddress}) => {
       expect(op.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY);
 
       const contract = await op.contract();
-      const symbolReturn = await contract.storage()
-      expect(JSON.stringify(symbolReturn)).toEqual(JSON.stringify(UnitValue))
+      console.log(contract.address);
+
+      const storage = await contract.storage();
+      expect(JSON.stringify(storage)).toEqual(JSON.stringify(UnitValue));
+
       const ticketDeposit = await contract.methods.default(txRollupAddress, 'tz4VHgLiRx5ZZjwU2QaybHc11EMJk3NcyvVc', '1').send();
```

Comment:
curious, since we are passing a tz4 address that lives in our txRollup. Does that mean we are using the same rollup node for all TORU tests?